app/views/planning_application/site_visits/show.html.erb

```diff
@@ -18,7 +18,7 @@
 <p class="govuk-body">Response created by: <%= @site_visit.created_by.name %></p>
 <p class="govuk-body">Response created at: <%= @site_visit.created_at.to_fs %></p>
 <% if @site_visit.decision? %>
-  <p class="govuk-body">Visited at: <%= @site_visit.visited_at&.to_fs %></p>
+  <p class="govuk-body">Visited at: <%= @site_visit.visited_at&.strftime("%-d %B %Y") %></p>
```

Comment:
How about this instead:
```suggestion
  <p class="govuk-body">Visited at: <%= @site_visit.visited_at&.to_date.to_fs %></p>
```

This way it's using the configured formats in `config/initializers/date_formatter.rb` and we can ensure consistency.

Reply:
I just looked and we're using at least 17 different date formats on top of configured ones, so maybe I'm worrying unnecessarily.

Reply:
Ah didn't realise that was what was happening 👍